langchain/vectorstores/base.py

```diff
@@ -116,6 +117,15 @@ def similarity_search_with_relevance_scores(
         """Return docs and relevance scores in the range [0, 1].
 
         0 is dissimilar, 1 is most similar.
+
+        Args:
+            query: input text
+            k: Number of Documents to return. Defaults to 4.
+            score_threshold: Optional, a floating point value between 0 to 1 to
```

Comment:
Would you mind clarifying in the doc-string that the filtering is happening client side?